mtp_api/apps/security/management/commands/update_security_profiles.py

```diff
@@ -45,64 +46,104 @@ def handle_update(self, batch_size, recreate):
             PrisonerProfile.objects.update_current_prisons()
 
     def handle_credit_update(self, batch_size):
-        new_credits = Credit.objects.filter(sender_profile__isnull=True).order_by('pk')
-        new_credits_count = new_credits.count()
-        if not new_credits_count:
-            self.stdout.write(self.style.SUCCESS('No new credits'))
+        # TODO Remove below function once the logs show that it consistently
```

Comment:
can't remove "legacy" function until bank transfers stop, right? maybe the TODO should say that explicitly